.circleci/config.yml

```diff
@@ -1,42 +1,18 @@
 version: 2.1
-
-# Define a job to be invoked later in a workflow.
 setup: true
 
 orbs:
-  continuation: circleci/continuation@0.1.2
-  python: circleci/python@1.0.0
-
-jobs:
-  create-config:
-    executor: python/default
-    steps: 
-      - checkout
-      - run: mkdir -p workspace
-      - python/install-packages:
-          args: PyYAML
-          pypi-cache: false
-      - run:
-          name: Create config
-          command: pipenv run python .circleci/generate-config.py
-      - run: mv .circleci/generated.yml workspace
-      - persist_to_workspace:
-          root: workspace
-          paths:
-            - generated.yml
-  setup:
-    executor: continuation/default
-    steps:
-      - attach_workspace:
-          at: /tmp/workspace
-      - continuation/continue:
-          configuration_path: /tmp/workspace/generated.yml # use newly generated config to continue
-
+  test-harness: govstack-working-group/testutils@1.0.0
```

Comment:
Please use v1.0.1. Can you also check if using @latest instead of fixed version is working? Updating each BB on new orb release can be time consuming.

Reply:
CircleCi rekomend to use fixed version - https://circleci.com/docs/orbs-faq/

<img width="946" alt="Zrzut ekranu 2023-03-1 o 12 44 07" src="https://user-images.githubusercontent.com/80507519/222130088-bc8ff263-9b91-4813-9f55-16f89fd6f1ed.png">

Reply:
Thanks, I don't think this should be dangerous in our scenario when all building blocks are aligned but let's stick to fixed version for now.